features/old/providers/settings.feature

```diff
@@ -6,14 +7,30 @@ Feature: Settings management
   Background:
     Given a provider is logged in
 
-  @javascript
   Scenario: Strong password setting
     And I go to the usage rules settings page
     When I check "Strong passwords"
     And I press "Update Settings"
     Then I should see the settings updated
-    And provider "foo.3scale.localhost" should have strong passwords enabled
+    And the provider should have strong passwords enabled
     When I uncheck "Strong passwords"
     And I press "Update Settings"
     Then I should see the settings updated
-    And provider "foo.3scale.localhost" should have strong passwords disabled
+    And the provider should have strong passwords disabled
+
+  Scenario: Account approval required checkbox is enabled
+    Given the provider has 1 account plan
+    When I go to the usage rules settings page
+    Then field "Account approval required" is not disabled
+
+  Scenario: Account approval required checkbox is disabled
+    Given the provider has multiple account plans
+    And the provider has "Account plans UI visible" set to "true"
```

Comment:
This steop is the only difference between this and previous scenario. How does this causes `Account approval required` to be disabled?

Reply:
Not just this step, but also the previous one: "the provider has multiple account plans"

Reply:
But yes, this behavior is extremely confusing, the logic comes from:
https://github.com/3scale/porta/blob/b89fa6f80bf6e5a99b08b651a1fc657127097f9f/app/views/sites/usage_rules/edit.html.slim#L29-L35

https://github.com/3scale/porta/blob/b89fa6f80bf6e5a99b08b651a1fc657127097f9f/app/models/settings.rb#L30-L36